typing/typecore.ml

```diff
@@ -6429,11 +6510,11 @@ and type_cases
           | None -> None
           | Some scond ->
               Some
-                (type_unpacks ext_env mode_local unpacks scond
+                (type_unpacks' ext_env mode_local unpacks scond
```

Comment:
Regardless of the refactoring above, I favor getting rid of `type_unpacks'`. It's just more transparent to readers to have the call to `snd` be here. Otherwise, I don't know what processing happens in `type_unpacks'` separately from `type_unpacks`.

Reply:
I agree, and I get rid of `type_unpacks` altogether in a stacked PR: https://github.com/ncik-roberts/ocaml-jst/pull/1

It's possible that this stacked PR should have just been included as part of this patch---I will think about doing so for the upstream PR.